doc/guide/pd/polydispersity.rst

```diff
@@ -168,7 +199,7 @@ The polydispersity is
 .. math:: p = \sigma / \bar x
 
 Note that larger values of PD might need larger number of points and $N_\sigma$.
-For example, at PD=0.7 and radius=60 |Ang|, Npts>=160 and Nsigmas>=15 at least.
+For example, at PD=0.7 and radius=60 |Ang|, Npts>=160 and $N_\sigma$>=15 at least.
 
```

Comment:
Should use Nsigmas here since that corresponds to the name used in the GUI.  Elsewhere it is used as a variable in an equation, and so it is appropriate to use LaTeX markup.